CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# 10.0.0-rc.2 "ferrous-photon" (2020-06-15)
+
+### material
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **benchmark:** fix license header and correct test descriptions ([#19596](https://github.com/angular/components/issues/19596)) ([0d266e2](https://github.com/angular/components/commit/0d266e2)) |
+| bug fix |  **benchmark:** fix license header, remove a11y import, and remove [@title](https://github.com/title) doc ([#19582](https://github.com/angular/components/issues/19582)) ([7d4fcd5](https://github.com/angular/components/commit/7d4fcd5)) |
```

Comment:
I don't think these `benchmark` changes should be in here.

Reply:
agreed - removed